source/includes/inkling2-reference/_constants.html.md

```diff
@@ -0,0 +1,61 @@
+# Constants
+
+This is the reference for the keyword **const**. 
+
+```inkling2--code
+const MaxVelocity = 100
+const MinVelocity = 0
+const Environment = "NewYorkCity"
+const DebugMode = true
+const DestinationPoint = [4.5, 2.7, 9.2]
+```
+
+Inkling supports named constants for all Inkling types. The name can be used wherever a value can appear.
+
+Constants make code easier to understand and modify because the constant name
+conveys the semantics of the constant. They document the logic of the program.
+
+Types can be specified for constants. If a type is omitted, it is inferred. If a type is specified, the compiler validates that the constant value matches the specified type.
+
+The constant value is specified as an expression that can make use of supported operators, numeric or string literals, and other constants. Cyclical constant references are not allowed.
+
+
+```inkling2--code
+# Valid constant declarations
+const NumericConst1: number = 100
+const NumericConst4: number<0 .. 2> = 3 * (5 / 2) - 7
```

Comment:
the C programmer in me wonders if integer division truncates when I read this. Maybe worth a `# = 0.5` addition